### ISSUE TITLE
Upgrade from C104 -> C105 on Android ARM x64 for 1.44.x - Nightly (Checking C105)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#24805]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers. We basically want to make sure that components are working as they regressed in C98 when we first merged into `master`.

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Rewards
    - [x] Wallet balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Wallet panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by LaurenWags** [comment]
Please use this to check the PR build for C105 prior to merging to master.

**Comment by Uni-verse** [comment]
Verified on `Samsung Galaxy S 21` using build version:

```
Brave	1.44.60 Chromium: 105.0.5195.37 (Official Build) canary (32-bit) 
Revision	5f67e9f258cc28ab970e57572290edf1863ee3c5-refs/branch-heads/5195@{#621}
OS	Android 12; Build/SP1A.210812.016
```

- [x] Clean profile 
- [x] Upgraded profile (1.44.59 C104 --> 144.60 C105)

Example | Example
-|- 
![screenshot-1661203409786](https://user-images.githubusercontent.com/17885425/186023047-2818e1f4-e027-45bc-9002-eb291d26d146.png) | ![screenshot-1661203507714](https://user-images.githubusercontent.com/17885425/186023061-924bf7a5-6df5-4bc6-93fe-76977eac4817.png)
